### ISSUE TITLE
mode: camera-streamer, WebRTC, set flip camera image level or flip, all are unresponsive.

### ISSUE THREAD

**Issue by skyaee** [mainsail-crew/mainsail#1485]
### Mainsail Version:

v2.6.1

### Browser:

Chrome

### Device:

Desktop PC

### Operating System:

macOS

### What happened?

mode: camera-streamer, WebRTC, set flip camera image level or flip, all are unresponsive.
<img width="703" alt="截屏2023-07-25 18 16 35" src="https://github.com/mainsail-crew/mainsail/assets/40803751/19500071-1ee5-4f62-91b1-4698c10eecc2">
<img width="703" alt="截屏2023-07-25 18 16 17" src="https://github.com/mainsail-crew/mainsail/assets/40803751/493cee08-5307-45e1-9406-f2a4c6d1afd2">

### What did you expect to happen instead?

Upgrade from v3 to v4

### How to reproduce this bug?

After reinstalling the system on Raspberry Pi

**Comment by meteyou** [comment]
thanks for the report! the bug was bigger than expected. it affected almost all webcam clients.

**Comment by R4toLoko** [comment]
The timelapse appears to have been affected also.
It usually shows a thumbnail of the last shot it took, right now it shows nothing like if the time-lapse was disabled, then it shows the recording at the end.
So atm you don't really now if it is running or not until the end.

I have forgotten to enable it in the past and this timelapse thumbnail helped to identify it, stop the print, start it again and voila.

**Comment by R4toLoko** [comment]
@meteyou this whole problem appears to be crowsnest doing, not mainsail.
I am still collecting more info but it seems like they release a poop update.

**Comment by meteyou** [comment]
@R4toLoko plus don't use other issues for different problems. Pls open a new issue, if you think there is a bug!

**Comment by meteyou** [comment]
@R4toLoko Yes, such things are important, and for this reason, issues should be separated so as not to be forgotten. Therefore, I ask you to create a new issue so that it will not be forgotten. This is to keep the overview.
